configure.py

```diff
@@ -609,6 +609,12 @@
     default=False,
     help='compile V8 with minimal optimizations and with runtime checks')
 
+parser.add_option('--load-internal-modules-from-disk',
```

Comment:
I think "internal" is a bit misleading. At first glance, I thought it meant just the files in `lib/internal`. Maybe `--load-core-modules-from-disk` would be a better name?